Sources/App/Setup/Config+Setup.swift

```diff
@@ -6,7 +6,7 @@ extension Config {
     // allow fuzzy conversions for these types
     // (add your own types here)
     Node.fuzzy = [Row.self, JSON.self, Node.self]
-    
+    self.addConfigurable(middleware: AppMiddleware.init, name: "app-middleware")
```

Comment:
May be move this to separate function `setupMiddleware`(like `setupRoutes` and `setupPreparations`)? 
Because there's could be new middleware in future and it looks prettier

Reply:
Fixed